source/openapi-admin-v3.yaml

```diff
@@ -7221,6 +7242,18 @@ components:
           type: object
           description: |-
             **Deprecated**
+        indexed_queryable_fields_names:
```

Comment:
We are in the process of browning this field out for what its worth (permissions) and we will likely want to remove this in a few weeks

Reply:
Thanks, can do! Is there a ticket related to `permissions` deprecation? So I can add to a ticket to remove the option

Reply:
Thank you!